### DISCUSSION TITLE
Dropdown selection validation

### DISCUSSION THREAD

**Discussion by me21** [zauberzeug/nicegui#2436] [Ideas / Feature Requests]
There is a validation feature for text input field. I thought it would be great to have a similar feature in dropdown selection component (and possibly other input controls). Thank you!

**Comment by falkoschindler** [comment]
Thanks for the suggestion, @me21!
I wasn't aware of [Quasar's QSelect](https://quasar.dev/vue-components/select) having a validation feature we could integrate.
It looks like we basically need to derive from `ValidationElement` and pass the `validation` argument.
Anyone up for a pull request? 🙂

**Comment by me21** [comment]
It's not so easy, I tried

```py
class ValidationSelect(nicegui.elements.select.Select, nicegui.elements.mixins.validation_element.ValidationElement):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
```

and then replace `ui.select` with `ValidationSelect` in my code and add `validation` argument, but I got

```
Select.__init__() got an unexpected keyword argument 'validation'
Traceback (most recent call last):
  File "C:\Users\azarubkin\Documents\Python Projects\edoc2\venv\Lib\site-packages\nicegui\events.py", line 406, in wait_for_result
    await result
  File "C:\Users\azarubkin\Documents\Python Projects\edoc2\dialog_form.py", line 108, in on_edit_form_clicked
    ValidationSelect({row.id: row.name for row in departments},
  File "C:\Users\azarubkin\Documents\Python Projects\edoc2\dialog_form.py", line 28, in __init__
    super().__init__(*args, **kwargs)
TypeError: Select.__init__() got an unexpected keyword argument 'validation'
```

If I swap the superclasses, I get no error, but no Select control is visible.

**Comment by falkoschindler** [comment]
I just created PR #2498.

**Reply by falkoschindler** [reply]
This feature has been released with version 1.4.14.
